### DISCUSSION TITLE
Templates not found

### DISCUSSION THREAD

**Discussion by TheSteveBurgess** [django-components/django-components#357] [Q&A] [answered]
Hi,

In the auto generated component it says 

"Templates inside `[your apps]/components` dir and `[project root]/components` dir will be automatically found"

That's not happening for me. I have to manually add the "[your apps]/components" file path to the to DIRS config option in TEMPLATES or i get a "TemplateDoesNotExist" error.

Am i missing something?

**Comment by EmilStenstrom** [comment]
Interesting. The code that does the autodiscovery is here: https://github.com/EmilStenstrom/django-components/blob/master/django_components/__init__.py#L22 - specifically line 22.

That file is run when that package is imported, typically as the package is added as to INSTALLED_APPS. Can you verify that that line is run in your case?

**Comment by TheSteveBurgess** [comment]
That line is definitely being run. 

Now i know that it "should" work I'll put aside some time to figure out what's going on.  I'd imagine it's got to be something specific to my setup.

**Reply by EmilStenstrom** [reply]
Thank you! :)

**Reply by JuroOravec** [reply]
@TheSteveBurgess Did you get things working? Has this been resolved? What was the issue?

**Comment by JuroOravec** [comment, marked as answer]
Please see https://github.com/EmilStenstrom/django-components/discussions/375#discussioncomment-10645261 for how app-level components are imported.
